### ISSUE TITLE
Apply basic permissions via "codeowners" file

### ISSUE THREAD

**Issue by ncksllvn** [department-of-veterans-affairs/vets-website#9281]
GitHub use a special file called "codeowners" in order to automate the assignment of code reviewers based on what files are affected by a pull request.

From team conversations, it sounds like the best approach for this to start is by assigning @bethpotts, @DanielleSoCompany, and @peggygannon as the codeowners of the `pages` directory, and myself (@ncksllvn) as the owner of other directories. The [README of the Interim CMS](https://github.com/department-of-veterans-affairs/vagov-content) now contains instructions so that as VA employees create GitHub accounts and request access to the Interim CMS, we can also add them to the codeowners file at that time.

CC: @WinishaSmith @chrisj-usds

**Comment by chrisj-usds** [comment]
@ncksllvn Does that mean that any of the above-named people can approve / publish? Or does it need to be all of them? Or something in between?

**Comment by ncksllvn** [comment]
@chrisj-usds Right now just one of the people listed (or anyone with edit access to the repository even if not listed in the codeowners file) needs to approve a pull request. We can increase that number though if we want to.

**Comment by ncksllvn** [comment]
I merged a `codeowners` file just now to get things started. Now, when a pull request is opened to any content page (the `pages/` directory), the content team is automatically assigned for review.  \

For example, after merging that change, I opened this PR - https://github.com/department-of-veterans-affairs/vagov-content/pull/223, which proposed a change to a content page -

![image](https://user-images.githubusercontent.com/1915775/51200613-03ee8580-18c8-11e9-9c9e-66ac395be8ab.png)

The content team was automatically notified as codeowners. Pretty cool!
